src/Tracing/Status.php

```diff
@@ -46,7 +46,7 @@ class Status
 
     private $code;
     private $description;
-    static private $ok;
+    private static $ok;
```

Comment:
My linter said that this was the correct way to instantiate this variable @morrisonlevi; I remember us looking at this during our pairing session.